optimade/utils.py

```diff
@@ -136,6 +140,13 @@ def get_child_database_links(
     links_endp = base_url + "/v1/links"
     try:
         links = requests.get(links_endp, timeout=10, headers=headers)
+    except SSLError as exc:
+        if skip_ssl:
+            links = requests.get(links_endp, timeout=10, headers=headers, verify=False)
+        else:
+            raise RuntimeError(
+                f"SSL error when connecting to provider {provider['id']}. Use `skip_ssl` to ignore."
```

Comment:
```suggestion
                f"SSL error when connecting to provider {provider['id']}. Use `--skip-ssl` to ignore."
```
Should this be the CLI option? Or do you not wish to mix CLI options and function parameters here? If not, it's fine to leave as is for sure.